packages/fluentui/react-bindings/src/hooks/usePrevious.ts

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+export const usePrevious = <ValueType = unknown>(value: ValueType, initialValue: ValueType | null = null) => {
+  const ref = React.useRef<ValueType | null>(initialValue);
```

Comment:
```suggestion
export const usePrevious = <ValueType = unknown>(value: ValueType, initialValue: ValueType | undefined = undefined): ValueType | undefined => {
  const ref = React.useRef<ValueType | undefined>(initialValue);
```

Let's use `undefined`, it's closer to other implementations in this repo 🙄